templates/auth/reset.html

```diff
@@ -35,12 +35,12 @@ <h5>Recover password</h5>
         </div>
         {% endif %}
         <div class="grid-x">
-            <div class="medium-12 columns">
+            <div class="medium-6 columns">
```

Comment:
Why modify this but not the rest?

Reply:
because if we change the other then register button and forget your password will come on same line.